runtime/src/hardened_unpack.rs

```diff
@@ -206,6 +206,7 @@ fn is_valid_snapshot_archive_entry(parts: &[&str], kind: tar::EntryType) -> bool
         (["accounts", file], GNUSparse) if like_storage(file) => true,
         (["accounts", file], Regular) if like_storage(file) => true,
         (["snapshots"], Directory) => true,
+        (["snapshots", "status_cache"], GNUSparse) => true,
         (["snapshots", "status_cache"], Regular) => true,
         (["snapshots", dir, file], Regular) if all_digits(dir) && all_digits(file) => true,
```

Comment:
Should we go ahead and allow here as well?

Reply:
yep, might as well. done